.golangci.yml

```diff
@@ -9,6 +9,16 @@ run:
 linters-settings:
   goimports:
     local-prefixes: antrea.io/antrea
+  revive:
+    ignore-generated-header: false
+    severity: warning
```

Comment:
what's the effect of this? Does golangci-lint still fail in case of warnings?

Reply:
yes, it will fail if there is any warnings, there are "warning" or "error" severity, I suppose we also want it fail if there was any code doesn't match the rules we enabled?